vmm/src/device_manager.rs

```diff
@@ -85,7 +85,7 @@ use vm_virtio::{DmaRemapping, IommuMapping, VirtioDeviceType, VirtioIommuRemappi
 use vm_virtio::{VirtioSharedMemory, VirtioSharedMemoryList};
 use vmm_sys_util::eventfd::EventFd;
 
-#[cfg(feature = "mmio_support")]
+#[cfg(any(feature = "mmio_support", target_arch = "aarch64"))]
```

Comment:
Why do you need this when mmio is not enabled?

Reply:
Even "mmio_support" feature is not enabled, aarch64 still have to add legacy devices (like serial https://github.com/cloud-hypervisor/cloud-hypervisor/blob/v0.8.0/vmm/src/device_manager.rs#L1387) to mmio bus, because IO bus is not an option. Feature "mmio_support" is used to control the transport option of virtio devices.